src/Sarif.Multitool/AbsoluteUriOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using CommandLine;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    [Verb("absoluteuri", HelpText = "Turn all relative Uris into absolute URIs (to be used after rebaseUri is run)")]
+    internal class AbsoluteUriOptions : MultitoolOptionsBase
+    {
+        [Value(0,
+            MetaName = "<files>",
+            HelpText = "Files to process (wildcards ? and * allowed).",
+            Required = false)]
+        public IList<string> Files { get; internal set; }
```

Comment:
>Files [](start = 29, length = 5)

Please rationalize all file argument command across driver framework and this code.